stretching/App/StretchApp.cs

```diff
@@ -143,6 +143,9 @@ private void OnDataLoad()
         private void PlotGraph()
         {
             //TODO: Plot graph
+
+
+
             throw new NotImplementedException();
```

Comment:
Here You should draw raph

Reply:
I know where :D I just put it there, coz I didnt know how to implement this here, and how to use parser data..